.github/ISSUE_TEMPLATE/crash.yml

```
@@ -0,0 +1,74 @@
+name: Crash report
+description: A hard crash of the interpreter, possibly with a core dump
+labels: ["type-crash"]
+body:
+  - type: markdown
+    attributes:
+      value: |
+          This form is for hard crashes of the Python interpreter, segmentation faults, failed C-level assertions, and similar.
+          Unexpected exceptions raised from Python functions in the standard library count as bugs rather than crashes.
+
+          The CPython interpreter is written in a different programming language, C.
+          A "CPython crash" is when Python itself fails, leading to a traceback in the C stack.
+  - type: dropdown
+    attributes:
+      label: CPython versions tested on
+      multiple: true
+      options:
+        - "3.8"
+        - "3.9"
+        - "3.10"
+        - "3.11"
+        - "3.12"
+        - "3.13 (main branch or an alpha release)"
+    validations:
+      required: true
+  - type: dropdown
+    attributes:
+      label: Operating systems tested on
+      multiple: true
+      options:
+        - Linux
+        - macOS
+        - Windows
+        - Other
+    validations:
+      required: false
+  - type: input
+    attributes:
+      label: Output from `python -VV`
+      description: (If you tested with multiple operating systems or architectures, feel free to provide details in the main bug description.)
+    validations:
+      required: false
+  - type: textarea
+    attributes:
+      label: What happened?
+      description: >
+        Include a [minimal, reproducible example](https://stackoverflow.com/help/minimal-reproducible-example) if possible.
+        Put any code blocks inside triple backticks.
+
+      value: |
+        ```python
+        # Add a code block here, if required
+        ```
+    validations:
+      required: true
+  - type: textarea
+    attributes:
+      label: Error messages
+      description: >
+        Enter any error messages caused by the crash, including a core dump if there is one.
+        Feel free to leave this bit blank if it isn't relevant.
+      placeholder: |
+        Error messages should be formatted like this:
+
+        <details>
+        <summary>Error messages/core dump</summary>
+
+        ```
+        [paste error here]
```

```suggestion
        # paste errors here, if you have any
```
You might want to use this for consistency with the other placeholder above.